### PR TITLE
Update heartbeat sample configuration files

### PR DESCRIPTION
**Pull request by urso**
- add banner to configuration files
- use a small sample config in heartbeat.yml only.

### CODE REVIEW

File: heartbeat/_meta/beat.yml [hunk 1]
```diff
@@ -1,191 +1,20 @@
 ################### Heartbeat Configuration Example #########################
 
+# This file is an example configuration file highlighting only some common options.
+# The heartbeat.full.file file in the same directory contains all the supported options
```
Comment (ruflin) [inline heartbeat/_meta/beat.yml]: heartbeat.full.yml (instead of file)

**Comment by urso** [comment]
> Left a minor comments. Besides that looks good to me. I think what is most important is that if some just runs the binary, it works out of the box and this seems to be the case with icmp only.

hmm... icmp might not work on windows though

File: heartbeat/_meta/beat.yml (hunk 1 above)
Comment (urso) [inline heartbeat/_meta/beat.yml]: fixed

File: heartbeat/_meta/beat.yml [hunk 2]
```diff
@@ -1,191 +1,20 @@
...
-
-  # Monitor name used for job name and document type.
-  #name: icmp
-
-  # Enable/Disable monitor
-  #enabled: true
-
-  # Configure task schedule using cron-like syntax
-  schedule: '*/5 * * * * * *' # exactly every 5 seconds like 10:00:00, 10:00:05, ...
+- type: icmp
```
Comment (tsg) [inline heartbeat/_meta/beat.yml]: I think you were saying the `icmp` is not available on Windows, right? I'd like the default configuration to work by default on all platforms. Plus I find the case of HTTP monitoring more natural for Heartbeat.

**Comment by tsg** [comment]
jenkins, package it

File: heartbeat/_meta/beat.yml (hunk 2 above)
Comment (ruflin) [inline heartbeat/_meta/beat.yml]: If we switch to http as default, the discussion comes up which port should be monitored by default. In the default setup we expect 9200 to be available (ES). Current default is 80.

Comment (monicasarbu) [inline heartbeat/_meta/beat.yml]: +1 for 9200 port by default.

Comment (urso) [inline heartbeat/_meta/beat.yml]: ok, will go with http + 9200 for minimal config

**Pull request merged**